src/funding/paylater/config.jsx

```diff
@@ -32,23 +20,42 @@ export function getPaylaterConfig() : FundingSourceConfig {
             BUTTON_LAYOUT.VERTICAL
         ],
 
-        Label: flexLabel,
+        Label: ({ logo }) => logo,
+
+        Logo: ({ logoColor, nonce, fundingEligibility }) => {
+            const paylaterEligibility = fundingEligibility.paylater;
 
-        Logo: flexLabel,
+            let text;
+
+            if (paylaterEligibility && paylaterEligibility.products && paylaterEligibility.products.payIn4 && paylaterEligibility.products.payIn4.eligible) {
+                text = <Text>Pay Later</Text>;
+            } else {
+                text = (
+                    <Fragment>
+                        <Text optional>PayPal </Text>
```

Comment:
This appears to be flickering when the browser is resized at the smaller split button breakpoint. For some pixel width values `PayPal Flex` shows, but for others only `Flex` shows when it looks like `PayPal Flex` should fit in the given space.

Reply:
Will check on this.

Reply:
Fixed